lib/mayaUsd/render/mayaToHydra/ProductionRender/ProductionSettings.cpp

```diff
@@ -520,6 +529,14 @@ std::string ProductionSettings::CreateAttributes()
 
 	std::string controlCreationCmdTemplate = "attrControlGrp -label \"%s\" -attribute \"%s.%s\";";
 
+	// Create attrbiutes for usd camera
+
+	_CreateBoolAttribute(node, MString(g_attributePrefix.GetText()) + "Static_useUSDCamera", false, userDefaults);
+
+	TfTokenVector vec;
+	TfToken token;
+	//_CreateEnumAttribute(node, MString(g_attributePrefix.GetText()) + "Static_usdCameraSelected", vec, token, userDefaults);
```

Comment:
Please remove if this line is not needed

Reply:
Done